docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,586 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hollweg Dispersion Solver\n",
+    "\n",
+    "[tfds]: ../../api/plasmapy.dispersion.numerical.hollweg_.hollweg.rst\n",
+    "[bellan2012]: https://doi.org/10.1029/2012JA017856\n",
+    "[hollweg1999]: https://doi.org/10.1029/1998JA900132\n",
+    "\n",
+    "This notebook details the functionality of the [hollweg()][tfds] function.\n",
+    "This function computes the wave frequencies for given wavenumbers and\n",
+    "plasma parameters based on the solution presented by [Bellan 2012][bellan2012]\n",
+    "to the [Hollweg 1999][hollweg1999] two fluid dispersion relation.\n",
+    "The dispersion relation assumes a uniform magnetic field, zero D.C\n",
+    "electric field, quasi-neutrality, wave propogation nearly perpendicular to\n",
+    "the magnetic field, a low-beta regime $c_{s} \\ll v_{A}$,\n",
+    "and low-frequency waves $\\omega \\ll \\omega_{ci}$ which equates to\n",
```

Comment:
```suggestion
    "This notebook details the functionality of the [hollweg()][tfds] function.  This function computes the wave frequencies for given wavenumbers and plasma parameters based on the solution to the two fluid dispersion relation presented by [Hollweg 1999][hollweg1999], and further summarized by [Bellan 2012][bellan2012].  In his derivation Hollweg assumed a uniform magnetic field, zero D.C electric field, quasi-neutrality, and low-frequency waves ($\\omega \\ll \\omega_{ci}$), which yielded the following expression\n",
```

I didn't change too much of the wording here, but there are two significant points...

1. In sentence 2, instead of giving credit to Bellan and then mentioning Hollweg reverse the statement.  Since Hollweg is the one that derived the expression, give him credit and then mention that Bellan expands on it.
2. When listing the assumptions, only list the ones that Hollweg used instead adding the two extra points Bellan made.  In one of the my comments bellow, we can make an explicit note about Bellan's additional restrictions.